lmod_proxy/edx_grades/actions.py

```diff
@@ -119,3 +129,13 @@ def get_sections(gradebook, form):
         data,
         not bool(error_message)
     )
+
+
+def _get_assignment_name_max_points(form):
+    try:
+        csv_data = form.datafile.data.stream.read()
+        short_name = csv_data[0][1]
+        max_points = csv_data[1][2]
```

Comment:
It looks like this function will handle only CSVs with one assignment since it assumes `max_points` is the third column